src/js/messaging/components/forms/NewMessageForm.jsx

```diff
@@ -65,6 +65,7 @@ export class NewMessageForm extends React.Component {
           allowedMimeTypes={allowedMimeTypes}
           errorMessage={validations.isValidMessageBody(message.body) ? undefined : composeMessage.errors.message}
           files={this.props.message.attachments}
+          loading={this.props.loading}
```

Comment:
I think you forgot to remove this, since you decided not to pass in the `loading` object anymore.

Reply:
Good call!